### PR TITLE
update json

### PR DESCRIPTION
**Pull request by FrancescoXX**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6251"></a>

### CODE REVIEW

**Review by khushalarora — APPROVED** [review]
Looks good, thanks :tada:

**Review by harshsingh32 — COMMENTED** [review]
Looks fine

File: data/FrancescoXX.json [hunk 1]
```diff
@@ -68,6 +68,12 @@
   "tags": [
     "Open Source",
     "Javascript",
+    "Typescript",
+    "JavaScript",
```
Comment (kumarsonsoff3) [inline data/FrancescoXX.json:72]: Hey Francesco, we don't need to use the same tag twice. Please remove the duplicate tag here. 🛠️ 

```suggestion
    "Javascript",
    "Typescript",
```

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**